probatus/feature_elimination/feature_elimination.py

```diff
@@ -354,14 +370,23 @@ def fit(self, X, y, column_names=None):
         if self.random_state is not None:
             np.random.seed(self.random_state)
 
+        if columns_to_keep is None:
+            columns_to_keep = []
+        
+
+
         self.X , self.column_names = preprocess_data(X, X_name='X', column_names=column_names, verbose=self.verbose)
         self.y = preprocess_labels(y, y_name='y', index=self.X.index, verbose=self.verbose)
         self.cv = check_cv(self.cv, self.y, classifier=is_classifier(self.clf))
 
         remaining_features = current_features_set = self.column_names
         round_number = 0
 
-        while len(current_features_set) > self.min_features_to_select:
+        if (self.min_features_to_select+len(columns_to_keep)) > len(self.column_names):
+            raise ValueError('Minimum features to select is greater than number of features.'
```

Comment:
Lets make it a warning, if verbose>50. This will make the feature a bit more usable.

Reply:
This  is a case, where the code should raise an error, since the number of columns to   select are greater than the number of columns in the data.In case we raise a warning, how would you handle the situation ?

Reply:
You are right :)